clang/lib/CodeGen/Targets/AArch64.cpp

```diff
@@ -761,8 +762,10 @@ bool AArch64ABIInfo::passAsPureScalableType(
       getContext().getBuiltinVectorTypeInfo(cast<BuiltinType>(Ty));
   assert(Info.NumVectors > 0 && Info.NumVectors <= 4 &&
          "Expected 1, 2, 3 or 4 vectors!");
-  auto VTy = llvm::ScalableVectorType::get(CGT.ConvertType(Info.ElementType),
-                                           Info.EC.getKnownMinValue());
+  llvm::Type *EltTy = Info.ElementType->isMFloat8Type()
```

Comment:
I am not sure when this is used? Can we have test for this?

Reply:
This is also covered by `clang/test/CodeGen/AArch64/pure-scalable-args.c`, but perhaps only after https://github.com/llvm/llvm-project/pull/118961
(this PR was originally on top of that PR, but I decoupled them on Spencer's request)
This code should trigger for `svmfloat8x2_t` and `svmfloat8x4_t`.